_posts/platform/app/2000-01-01-review-apps.md

```diff
@@ -40,6 +40,40 @@ As a quick example, here is a sample `scalingo.json` that customize the environm
 
 Here, the value of the environment variable `CANONICAL_HOST_URL` will contain the URL to reach the newly deployed app. The `scalingo.json` configuration always takes precedence on parent app configuration.
 
+## Run a task after the 1st deployment of a review app
+
+When a review app is created, you might want to execute a custom action like seeding its database, this can be done using the `first-deploy` scripts in the manifest `scalingo.json`. If such a property is defined, the given command will be executed as a [postdeploy hook]({% post_url platform/app/2000-01-01-postdeploy-hook %}) after the first deployment.
+
+{% warning %}
+This script replaces the postdeploy hook. If you want to execute both, make sure to adapt the `first-deploy` has the same effect that your `postdeploy` additionally to the special action you want to achieve at the first deployment.
```

Comment:
I don't understand the second sentence of this paragraph, starting from `If you want to execute both....`.